modules/debugging/responses.js

```diff
@@ -0,0 +1,45 @@
+import { BANNER, NATIVE, VIDEO } from '../../src/mediaTypes.js';
+
+export default {
+  [BANNER]: () => `<html><head><body><img src="https://vcdn.adnxs.com/p/creative-image/27/c0/52/67/27c05267-5a6d-4874-834e-18e218493c32.png"/></body></html>`,
+  [VIDEO]: () => '<?xml version=\"1.0\" encoding=\"UTF-8\"?><VAST version=\"3.0\"><Ad><InLine><AdSystem>GDFP</AdSystem><AdTitle>Demo</AdTitle><Description><![CDATA[Demo]]></Description><Creatives><Creative><Linear ><Duration>00:00:11</Duration><VideoClicks><ClickThrough><![CDATA[https://prebid.org/]]></ClickThrough></VideoClicks><MediaFiles><MediaFile delivery=\"progressive\" width=\"640\" height=\"360\" type=\"video/mp4\" scalable=\"true\" maintainAspectRatio=\"true\"><![CDATA[https://s3.amazonaws.com/files.prebid.org/creatives/PrebidLogo.mp4]]></MediaFile></MediaFiles></Linear></Creative></Creatives></InLine></Ad></VAST>',
+  [NATIVE]: () => ({
+    ortb: {
+      link: {
```

Comment:
your approach here is to fill every possible asset type, which works for "legacy" native. 

We recommend using ortb now though, and in it you're allowed for example to ask for more than one image. Also, templates written for ORTB native won't necessarily work with this.

What this should do IMO is loop over `bidRequest.nativeOrtbRequest.assets` and provide a default value for each asset. [Here's a reference](https://www.iab.com/wp-content/uploads/2018/03/OpenRTB-Native-Ads-Specification-Final-1.2.pdf) (look for "Asset Response").